setup.py

```diff
@@ -55,7 +55,14 @@ def build_cython():
         if os.path.exists(pyx) and needs_compilation(c, pyx)]
     if len(nc_pyx_filenames) > 0:
         cmd = ['cython'] + nc_pyx_filenames
-        subprocess.check_call(cmd)
+        try:
+            subprocess.check_call(cmd)
```

Comment:
@LeeKamentsky I'm not totally clear on the doc string.  Is there a reason we don't add `cython` to the `setup_requires` and use that copy?  I would think that would be a good default and we could add a mechanism to force usage of another Cython instance if really needed.

Reply:
I think adding cython to setup_requires is a good thing - it can't hurt.